CMakeLists.txt

```
@@ -170,6 +176,14 @@ if(CMAKE_BUILD_TESTS)
             "No Python virtual environment detected. Please activate one.")
   endif()
 
+  if(NOT "${GPU_DEVICE}" STREQUAL "NONE")
+    # If a GPU device is passed, specify if more than one is available. Set to
+    # OFF if only one is available. Used to determine whether to run Multi GPU
+    # testing
+    option(MULTI_GPU "Whether to enable Multi GPU testing (ON [default], OFF)"
+           OFF)
```

Should this not be set to `ON` as the default?
```suggestion
           ON)
```

---

You'll then need to amend the workflow to pass `-DMULTI_GPU=OFF`